src/dlgConnectionProfiles.cpp

```diff
@@ -1547,17 +1547,18 @@ void dlgConnectionProfiles::loadProfile(bool alsoConnect)
     if (entries.isEmpty()) {
         firstTimeLoad = true;
     } else {
-        QString fileName{qsl("%1%2").arg(folder, profile_history->itemData(profile_history->currentIndex()).toString())};
-        QFile file(fileName);
+        QFile file(qsl("%1%2").arg(folder, profile_history->itemData(profile_history->currentIndex()).toString()));
         file.open(QFile::ReadOnly | QFile::Text);
         XMLimport importer(pHost);
 
         qDebug() << "[LOADING PROFILE]:" << file.fileName();
         if (auto [success, message] = importer.importPackage(&file, nullptr); !success) {
             pHost->postMessage(tr("[ ERROR ] - Something went wrong loading your Mudlet profile and it could not be loaded.\n"
-                "Try loading an older version in 'Connect - Options - Profile history' or double-check that %1 looks correct.", "%1 is the filename").arg(fileName));
+                "Try loading an older version in 'Connect - Options - Profile history' or double-check that %1 looks correct.").arg(file.fileName()));
         
-            qDebug() << "dlgConnectionProfiles::loadProfile: ERROR loading" << fileName << "due to:" << message;
+            qDebug() << "dlgConnectionProfiles::loadProfile: ERROR loading" << file.fileName() << "due to:" << message;
```

Comment:
```suggestion
            qDebug() << "dlgConnectionProfiles::loadProfile: ERROR loading " << file.fileName() << " due to: " << message;
```

Reply:
That isn't needed - because `qDebug::operator<<()` automatically adds a space separator (and double quotation marks) around each item in the list by default (or unless it was told to behave differently) - which is why I tend to be explicit and I would have said:
```suggestion
            qDebug().nospace().noquote() << "dlgConnectionProfiles::loadProfile(" << alsoConnect << ") ERROR - loading \"" << file.fileName() << "\" failed, reason: \"" << message << "\".";
```